intermediate_source/DCP_tutorial.rst

```diff
@@ -0,0 +1,241 @@
+Getting Started with Distributed Checkpoint (DCP)
+=====================================================
+
+**Author**: `Iris Zhang <https://github.com/wz337>`__, `Rodrigo Kumpera <https://github.com/kumpera>`__, `Chien-Chin Huang <https://github.com/fegin>`__
+
+.. note::
+   |edit| View and edit this tutorial in `github <https://github.com/pytorch/tutorials/blob/main/intermediate_source/DCP_tutorial.rst>`__.
+
+Checkpointing AI models during distributed training could be challenging, as parameters and gradients are partitioned across GPUs and the number of GPUs available could change when you resume training.
```

Comment:
Should we change GPUs to trainers? Users may not necessarily use GPUs but still need DCP.